### DISCUSSION TITLE
termux-url-opener

### DISCUSSION THREAD

**Discussion by backtabber** [termux/termux-app#3462] [Q&A] [answered]
Please tell me what I need to do differently.

1. I created the file $HOME/bin/termux-url-opener
2. It contains: echo script was triggered > test

I shared a url from my browser to termux, but when I look in the bin directory, there is no file named test.

**Comment by agnostic-apollo** [comment, marked as answer]
The `test` file should be created in `$HOME` since that would be the current working directory of the shell started, check `pwd` output in script.
